model/schema.js

```diff
@@ -1,26 +1,24 @@
-const mongoose = require('mongoose');
 const express = require('express');
-
+const mongoose = require('mongoose');
+const bcrypt= require('bcrypt');
 
 const  clientSchema = new mongoose.Schema({
 name:{
    type:String,
    required:true 
 },
-phoneno:{
- type:Number,
-validator:{
-    Length:10
-}} ,
 email:{
     type:String,
     required:true
 },
-password:{
+createpassword:{
```

Comment:
This scheme should be change to password.
```suggestion
password:{
```